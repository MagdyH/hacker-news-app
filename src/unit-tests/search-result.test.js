import React from 'react';
import renderer from 'react-test-renderer';
import SearchResult from '../components/search-result';
import configureStore from 'redux-mock-store';
import { mount } from 'enzyme';
import { Provider } from "react-redux";

const initialState = {}
const mockStore = configureStore();
const store = mockStore(initialState);


describe('Search Result component', () => {
    it('should render snapshot', () => {
        const component = renderer.create(<SearchResult store={store} />);

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});