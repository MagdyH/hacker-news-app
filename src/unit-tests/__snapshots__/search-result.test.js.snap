// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Result component should render snapshot 1`] = `
<section
  className="Search-Results"
>
  <div
    className="infinite-scroll-component__outerdiv"
    style={Object {}}
  >
    <div
      className="infinite-scroll-component "
      style={
        Object {
          "WebkitOverflowScrolling": "touch",
          "height": "auto",
          "overflow": "auto",
        }
      }
    >
      <div
        className="loader"
      >
        Loading ...
      </div>
    </div>
  </div>
</section>
`;
